Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -622,6 +622,18 @@ def test_create_subprocess_shell_text_mode_fails(self):
         self.loop.run_until_complete(execute())
 
 
+    def test_create_subprocess_exec_with_path(self):
+        async def execute():
+            from pathlib import Path
```

Comment:
Move the import to the top of the module.
Use `import pathlib` form and `pathlib.Path` for consistency.